packages/ui/src/components/va-affix/VaAffix.stories.ts

```diff
@@ -1,12 +1,51 @@
-import { defineComponent } from 'vue'
-import VaAffix from './VaAffix.demo.vue'
+import { VaAffix } from './'
+import { expect } from '@storybook/jest'
 
 export default {
   title: 'VaAffix',
   component: VaAffix,
 }
 
-export const Default = defineComponent({
+export const FixedTop = () => ({
   components: { VaAffix },
-  template: '<VaAffix/>',
+  template: `
+    <div class="w-1/4">
+      <p class="m-1">{{$vb.lorem(100)}}</p>
+        <VaAffix :offset-top="100">
+          <div class="p-3 bg-red-500">Fixed at the top: 100</div>
+        </VaAffix>
+      <p class="m-1">{{$vb.lorem(2500)}}</p>
+    </div>
+  `,
 })
+
+FixedTop.play = async ({ step }) => {
```

Comment:
Noticed we have this test failing for some reason.

<img width="778" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/6d11d7e4-a363-4df5-a5e1-18ba939f3945">